### PR TITLE
refactor: use the authorization header in the healthcheck

### PR DESCRIPTION
**Pull request by GuillaumeTech**
transfer the authorization header to the graphQL healthcheck call

- [ ] I wrote tests for the feature
- [ ] I documented the feature if needed

If the feature is a refactor, please explain why your way is better

### CODE REVIEW

File: botfront/imports/startup/server/apollo.js [hunk 1]
```diff
@@ -51,7 +51,12 @@ export const runAppolloServer = () => {
     });
 
     WebApp.connectHandlers.use('/health', (req, res) => {
-        axios.get('http://localhost:3000/graphql?query=query%20%7BhealthCheck%7D').then((response) => {
+        const { authorization } = req.headers;
+        axios.get('http://localhost:3000/graphql?query=query%20%7BhealthCheck%7D', {
+            headers: {
```
Comment (znat) [inline botfront/imports/startup/server/apollo.js]: Shouldn't we handle the case where authorization is undefined and just not set headers in that case?

File: botfront/imports/startup/server/apollo.js [hunk 2]
```diff
@@ -51,7 +51,12 @@ export const runAppolloServer = () => {
     });
 
     WebApp.connectHandlers.use('/health', (req, res) => {
-        axios.get('http://localhost:3000/graphql?query=query%20%7BhealthCheck%7D').then((response) => {
+        const { authorization } = req.headers;
```
Comment (znat) [inline botfront/imports/startup/server/apollo.js]: How the API_KEY env value is made available as `req.headers`?

Comment (GuillaumeTech) [reply botfront/imports/startup/server/apollo.js]: The authorization headers take the API_KEY value in the liveness probe headers, see this commit : https://github.com/botfront/botfront-helm/pull/1/commits/e604694bdfc87bca5cba7a56484bd8d85bd57733

File: botfront/imports/startup/server/apollo.js (hunk 1 above)
Comment (GuillaumeTech) [reply botfront/imports/startup/server/apollo.js]: Yes, I've made the change

**Pull request merged**